### ISSUE TITLE
slicing within `select()`

### ISSUE THREAD

**Issue by nickteff** [has2k1/plydata#9]
Is it possible to slice with the `select()` verb?  

For example, suppose I have a dataframe

```python
df = pd.DataFrame({"a": range(3), "x" : range(3), 
                   "y": range(3), "z": range(3)}, 
                   index=range(3))
```

I can do this

```python
df >> select("x", "y", "z")
```

Is it possible in someway to do this via a slice `"x":"z"`?  I tried

**edited**
```python
df >> select("x":"z") #corrected typo here "z" was "y"
```
and

**edited**
```python
df >> select(("x":"z")) # corrected typo here "z" was "y"
```
with no success.

**Comment by has2k1** [comment]
Python syntax restrictions mean that ` select("x":"y")` and  `select("x":"y")` are impossible. Maybe implement `select(slice("x", "y"))`. Does that make sense?

**Comment by nickteff** [comment]
Not quite.  Can you elaborate on what you are thinking with implementing `select(slice("x", "z"))`.  I need a little more direction.

Note:  I edited my comment to correct the `"x":"y"` typos.

**Comment by has2k1** [comment]
You can only do `'x':'y'` as a key/index. i.e. it has to be of the form obj['x':'y'], then it is turned into a slice, otherwise it is a syntax error.

**Comment by nickteff** [comment]
Thank you for the feedback.  I can research it a bit myself, but if you have some pointers to help one get a head start, then they would be appreciated.

**Comment by has2k1** [comment]
I see the confusion. I mean, plydata implementing the `slice` function. It is not possible now, but it is good feature. I'll add it may be tomorrow and make a release.

**Comment by has2k1** [comment]
Released in v0.3.2. You can now do `select(..., slice("x", "y"), ...)`.
